Kitodo/src/main/java/org/kitodo/production/services/data/FilterService.java

```diff
@@ -157,11 +157,6 @@ public BoolQueryBuilder queryBuilder(String filter, ObjectType objectType, Boole
 
         BoolQueryBuilder query = new BoolQueryBuilder();
 
-        // this is needed if we filter processes
```

Comment:
Why this part is removed?

Reply:
Because this is now done implicitly because the ProcessServic is extending the projectSearchService